src/main/java/net/pms/database/TableFilesStatus.java

```diff
@@ -300,21 +301,20 @@ private static void upgradeTable(final Connection connection, final int currentV
 	}
 
 	/**
-	 * Must be called in inside a table lock
+	 * Must be called from inside a table lock
 	 */
 	private static void createFilesStatusTable(final Connection connection) throws SQLException {
-		LOGGER.debug("Creating database table if it doesn't exist: \"{}\"", TABLE_NAME);
+		LOGGER.debug("Creating database table: \"{}\"", TABLE_NAME);
 		try (Statement statement = connection.createStatement()) {
 			statement.execute(
-				"CREATE TABLE IF NOT EXISTS " + TABLE_NAME + "(" +
+				"CREATE TABLE " + TABLE_NAME + "(" +
 					"ID            IDENTITY PRIMARY KEY, " +
 					"FILEID        INT      NOT NULL UNIQUE, " +
 					"MODIFIED      DATETIME, " +
 					"ISFULLYPLAYED BOOLEAN  DEFAULT false" +
 				")"
 			);
 
-			statement.execute("CREATE INDEX ID_IDX ON " + TABLE_NAME + "(ID)");
 			statement.execute("CREATE INDEX FILEID_IDX ON " + TABLE_NAME + "(FILEID)");
```

Comment:
The primary key always has an "implied index" so there's no need to create one explicitly.